sale_order_line_price_history/wizards/sale_order_line_price_history.py

```diff
@@ -109,4 +109,7 @@ class SaleOrderLinePriceHistoryline(models.TransientModel):
     @api.multi
     def action_set_price(self):
         self.ensure_one()
-        self.history_sale_order_line_id.price_unit = self.price_unit
+        self.history_sale_order_line_id.write({
```

Comment:
Put in on a hook for possible inheritance (for example, `sale_triple_discount`).